felix/bpf/ut/bpf_prog_test.go

```diff
@@ -394,25 +394,9 @@ func setupAndRun(logger testLogger, loglevel, section string, rules *polprog.Rul
 		defer o.Close()
 	}
 
-	log.Infof("Patching binary %s", obj+".o")
+	obj += ".o"
 
-	bin, err := bpf.BinaryFromFile(obj + ".o")
-	Expect(err).NotTo(HaveOccurred())
-	// XXX for now we both path the mark here and include it in the context as
-	// well. This needs to be done for as long as we want to run the tests on
-	// older kernels.
-	bin.PatchSkbMark(skbMark)
```

Comment:
can you please remove https://github.com/projectcalico/calico/blob/master/felix/bpf/binary.go#L93